src/raizes.py

```diff
@@ -15,3 +15,61 @@ def bhaskara(a, b, c):
         x1 = (-b + delta ** (1 / 2)) / (2 * a) 
         x2 = (-b - delta ** (1 / 2)) / (2 * a)
         print(f"x1: {x1}, x2: {x2}")
+
+
+
+
+def formatarComplexResult(realPart, imPart):
+	#um caso simples
+	if (realPart == 0 and imPart == 0):
+		return "0"
+	number = ""
+	if (realPart != 0):
+		number += str(realPart)
+	if (imPart > 0):
+		number += "+" + str(imPart) + "i"
+	elif (imPart < 0):
+		number += str(imPart) + "i"
+	else:
+		number += str(imPart) + "i"    
+	return number
+
+
+#Computa a(s) raíz(es) de um polinômio de terceiro grau usando o método de Tartaglia-Cardano:
+def cardano(a,b,c,d):	
+  if a == 0:
```

Comment:
```suggestion
import numpy as np
  if a == 0:
```

Você esqueceu de importar o `numpy` como `np`